src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -516,6 +516,8 @@
 
 "Home_Risk_Period_Update_Title" = "Updated every %@\U00A0hours";
 
+"Home_Risk_Period_Disabled_Button_Title" = "Updated only every %@\U00A0H";
```

Comment:
strings dict? :)

Reply:
Do you mean for **0** case ? I think, there will not be **0**, we always will have h > 0